mod-inventory/src/main/resources/domain/mod-inventory/features/inventoryFeatureTest.feature

```diff
@@ -50,3 +50,180 @@ Feature: inventory sample
       """
       When method POST
       Then status 201
+
+    Scenario: duplicate Instance,Holdings,Item creation
+#     First Instance
+      Given path 'inventory/instances'
+      And request
+      """
+      {
+        "source":"FOLIO",
+        "title":"TestInstance",
+        "instanceTypeId":"6312d172-f0cf-40f6-b27d-9fa8feaf332f"
+      }
+      """
+      When method POST
+      Then status 201
+      * def instancelocation = responseHeaders['Location'][0]
+      * def instanceId = instancelocation.substring(instancelocation.lastIndexOf('/') + 1)
+
+#     Duplicate Instance
+      Given path 'inventory/instances'
+      And request
+      """
+      {
+        "source":"FOLIO",
+        "title":"TestInstance",
+        "instanceTypeId":"6312d172-f0cf-40f6-b27d-9fa8feaf332f"
+      }
+      """
+      When method POST
+      Then status 201
+      * def dupInstancelocation = responseHeaders['Location'][0]
+      * def dupInstanceId = dupInstancelocation.substring(dupInstancelocation.lastIndexOf('/') + 1)
+#     Both instances ids should not match
+      * match dupInstanceId != instanceId
```

Comment:
Why is the instance being duplicated when the description specifically refers to duplicating only a holdings?

Reply:
@marcjohnson-kint :
We have Tasks for Instance & item also. 
i put that just to refer that if we use UUID function to generate ID , that wont justify the TASK .